src/runtime/entitlements-manager.js

```diff
@@ -234,7 +234,13 @@ export class EntitlementsManager {
   possiblyPingbackOnClientEvent_(event) {
     // Verify GAA params are present, otherwise bail since the pingback
     // shouldn't happen on non-metering requests.
-    if (!queryStringHasFreshGaaParams(this.win_.location.search)) {
+    // We don't validate access type since not all publishers use it.
```

Comment:
Could this comment read "We always want to send pingbacks" ?

Reply:
Added some more info along these lines.